app/src/main/java/org/oppia/app/player/state/StateFragmentPresenter.kt

```diff
@@ -611,4 +615,13 @@ class StateFragmentPresenter @Inject constructor(
   }
 
   private fun isAudioShowing(): Boolean = viewModel.isAudioBarVisible.get()!!
+
+  fun updateSubmitButton(
+    pendingAnswerError: String?
+  ) {
```

Comment:
Nit: Update this to single line -> `fun updateSubmitButton(pendingAnswerError: String?) {`

Reply:
updated